src/AWS/Kinesis/Akka.Streams.Kinesis/KinesisSourceStage.cs

```diff
@@ -202,7 +202,7 @@ private void Ready((IActorRef actorRef, object message) args)
 
             protected override void OnTimer(object timerKey)
             {
-                if ("GETRECORDS" == (string)timerKey)
+                if ("GET_RECORDS" == (string)timerKey)
```

Comment:
Can we stick this into a `const` that is shared between both places?

Reply:
Sure!